### PR TITLE
Adding xsens_official to documentation index for kinetic

### PR DESCRIPTION
**Pull request by Steven-GH**

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -13875,6 +13875,16 @@ repositories:
       url: https://github.com/ethz-asl/ethzasl_xsens_driver.git
       version: master
     status: maintained
+  xsens_official:
```
Comment (tfoote) [inline kinetic/distribution.yaml]: For consistency please have the name match the repository name so it's easier to find and match.

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -13875,6 +13875,16 @@ repositories:
...
+    doc:
+      type: git
+      url: https://github.com/xsens/xsens_mti_ros_node.git
```
Comment (tfoote) [inline kinetic/distribution.yaml]: Related i'll note that this repository name is a little misleading as it's more than just the ROS node.

**Review by tfoote — CHANGES_REQUESTED** [review]
Thanks for submitting this. Please at least update the top level entry so it matches the repository name. It's not necessary, but I would suggest considering a different name for the repository, something like `xsense_support` or possibly `xsense_driver_official` then make the top level key match that updated name.

File: kinetic/distribution.yaml (hunk 2 above)
Comment (StevenXsens) [reply kinetic/distribution.yaml]: Thank you for the feedback. Should it be named driver instead of node? Or do you suggest something else?

Comment (tfoote) [reply kinetic/distribution.yaml]: Often the repository will match the package name, but that's generally only done if there's only one package in the repository. I'd suggest `xsens_support` or something else similar. Or `xsens_ros_driver` maybe.

**Comment by dirk-thomas** [comment]
@StevenXsens Closing due to no response.

**Pull request closed without merging**